shared/bus.go

```diff
@@ -124,43 +123,43 @@ func CreateBusWithOptionalModules(
 	return bus
 }
 
-func (m *bus) PublishEventToBus(e *types.PocketEvent) {
+func (m bus) PublishEventToBus(e *debug.PocketEvent) {
```

Comment:
Why is `bus` no longer a pointer. Here and bellow?

Reply:
See Allesandro's `leak comment` above

Reply:
Noting that you probably meant @adshmh rather than @deblasis for diligence, but understood.